pkg/provider/aws/action/openshift-snc/openshift-snc.go

```diff
@@ -251,7 +265,29 @@ func (r *openshiftSNCRequest) deploy(ctx *pulumi.Context) error {
 }
 
 // Write exported values in context to files o a selected target folder
-func manageResults(stackResult auto.UpResult, prefix *string) error {
+func (r *openshiftSNCRequest) manageResults(stackResult auto.UpResult, prefix *string) (*OpenshiftSncResultsMetadata, error) {
```

Comment:
If this is shared may move it to a common package and reuse it within kind and openshift-snc

Reply:
ok..